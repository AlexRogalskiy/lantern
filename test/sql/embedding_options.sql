SET enable_seqscan = off;

CREATE EXTENSION IF NOT EXISTS vector;
CREATE EXTENSION IF NOT EXISTS lanterndb;

\qecho
\set ON_ERROR_STOP on


CREATE TABLE t (val vector(3));
-- todo::

DROP TABLE t;
