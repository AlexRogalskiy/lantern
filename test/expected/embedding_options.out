SET enable_seqscan = off;
SET
CREATE EXTENSION IF NOT EXISTS vector;
CREATE EXTENSION
CREATE EXTENSION IF NOT EXISTS lanterndb;
CREATE EXTENSION

CREATE TABLE t (val vector(3));
CREATE TABLE
DROP TABLE t;
DROP TABLE
